lib/options.js

```diff
@@ -40,8 +41,18 @@ function normalizeOptions(options) {
 	options.karma = options.karma || karma;
 	options.lintGlobs = options.lintGlobs || codeGlobs;
 	options.uglifySrc = options.uglifySrc || 'build/**/*.js';
-	options.moduleName = options.moduleName || 'metal';
+	options.moduleName = options.moduleName || packageJSON.name || 'metal';
 	options.noSoy = options.noSoy || false;
+	options.sauceLabs = Object.assign({
+		connectOptions: {
+			port: 5757,
+			logfile: 'sauce_connect.log'
+		},
+		startConnect: true,
+		recordScreenshots: false,
+		recordVideo: false,
+		testName: options.moduleName || ''
```

Comment:
This already has a default, so it will, at the very least, be `metal` if nothing was set above, right?